jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -231,6 +234,14 @@ class CawsConnectionProvider : GatewayConnectionProvider {
                                     }
                                 }
                             }
+                            CodecatalystTelemetry.devEnvironmentWorkflowStatistic(
+                                project = null,
+                                userId = userId,
+                                result = if (attemptCount == 15) TelemetryResult.Failed else TelemetryResult.Succeeded,
+                                duration = (System.currentTimeMillis() - start).toDouble(),
+                                codecatalystDevEnvironmentWorkflowStep = "fileSystem Check",
+                                numberOfRetryAttempts = (attemptCount).toString()
```

Comment:
```suggestion
                                numberOfRetryAttempts = attemptCount.toString()
```

Reply:
why not an int?

Reply:
Actually, types { } in telemetryOverride.json file was default set as "String". However, will remove this filed and add ```value``` instead